cypress/fixtures/navigationWhenLoggedIn.json

```diff
@@ -0,0 +1,20 @@
+{
+    "organizationNavigation": {
+        "siteLogo": ".vs-c-site-logo",
+        "searchButton": ".vs-c-project-search",
+        "notificationButton": ".vs-c-notification",
+        "howItWorksButton": ".vs-l-project__options > .vs-c-btn"
+    },
+    "boardNavigation": {
+        "siteLogo": ".vs-c-site-logo",
```

Comment:
You have `siteLogo` here and in `oranizationNavigation` too. Is it necessary to duplicate?